cassandra-3.11/build.sh

```diff
@@ -4,6 +4,7 @@ set -euo pipefail
 
 if [ -z ${1+x} ]; then 
```

Comment:
Unrelated: why not use `bash` syntax? I think `bash` is available everywhere these days and it is usually more intuitive in behaviour :)

Reply:
Not sure I follow. Feel free to add a suggestion and we can add it. I just want to check if an arg is set, no strong feelings on how that's accomplished.

Reply:
I meant using `bash` expressions in general. It is not related to this PR or to this particular line at all :)